src/SFA.DAS.Reservations.Api/StartupExtensions/ServiceCollectionExtensions.cs

```diff
@@ -19,5 +23,14 @@ public static IServiceCollection AddNServiceBusClientUnitOfWork(this IServiceCol
                 .AddScoped<IUnitOfWork, UnitOfWork.NServiceBus.Features.ClientOutbox.Pipeline.UnitOfWork>()
                 .AddScoped<IUnitOfWorkManager, UnitOfWorkManager>();
         }
+
+        public static void AddElasticSearch(this IServiceCollection collection, ReservationsConfiguration configuration)
+        {
+            var connectionPool = new  SingleNodeConnectionPool(new Uri(configuration.ElasticSearchServerUrl));
```

Comment:
Does this need to connect differently? Using the basic auth?
```suggestion
            var connectionPool = new  SingleNodeConnectionPool(new Uri(configuration.ElasticSearchServerUrl));
            
            var settings = new ConnectionSettings(connectionPool);
            
            if (!string.IsNullOrEmpty(configuration.ElasticSearchUsername) &&
                !string.IsNullOrEmpty(configuration.ElasticSearchPassword))
            {
                settings.BasicAuthentication(configuration.ElasticSearchUsername, configuration.ElasticSearchPassword);
            }
            

            collection.AddTransient<IElasticClient>(sp => new ElasticClient(settings));
```

Reply:
Yes at the time i wrote that part it was clear how we were going to connect to the AT and beyond environments and I missed updated this afterwards so will change this now

Reply:
💥